src/components/PageHeader/WalletPopup/SwapLoading/index.tsx

```diff
@@ -0,0 +1,151 @@
+import LoadingIndicator from 'assets/ui/loading-indicator.svg'
+import Button from 'components/Button'
+import { formatUnits } from 'ethers/lib/utils'
+import { trim } from 'helpers/trim'
+import { useTranslation } from 'next-i18next'
+import Image from 'next/image'
+import styled, { keyframes } from 'styled-components/macro'
+import { ContentLarge, ContentSmall, Headline } from 'styles/typography'
+import { SwapTransactionState, TokenInfo } from '../1inchHelper'
+import ArrowRight from './arrow-right.svg'
+
+interface Props {
+  swapState: SwapTransactionState
+  fromTokenInfo: TokenInfo
+  toTokenInfo: TokenInfo
+  onClose: () => void
+  onSuccess: () => void
+}
+
+const StyledLoadingContainer = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: ${({ theme }) => theme.colors.white};
+  padding: 20px 12px 12px 12px;
+  gap: 20px;
+`
+
+const StyledLoadingTitle = styled(ContentLarge).attrs({ as: 'h2' })``
+
+const StyledLoadingDesc = styled(ContentSmall).attrs({ as: 'p' })``
+
+const StyledSuccessBody = styled.div`
+  display: flex;
+  padding: 10px 0;
+  gap: 15px;
+`
+
+const Spin = keyframes`
+  0%, 10% {
+    transform: perspective(400px);
+  }
+  90%, 100% {
+    transform: perspective(400px) rotateY(-180deg);
+  }
+`
+
+const StyledCoin = styled.div`
+  width: 60px;
+  height: 60px;
+  animation: ${Spin} 3s cubic-bezier(0.3, 2, 0.4, 0.8) infinite both;
+  transform-style: preserve-3d;
+  position: relative;
+`
+
+const StyledCoinFace = styled.div`
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  background: #b5c0d0;
+  border-radius: 50%;
+
+  &:nth-child(1) {
+    transform: translateZ(-0.2em) rotateY(180deg);
+  }
+  &:nth-child(2) {
+    transform: translateZ(-0.1em);
+  }
+  &:nth-child(4) {
+    transform: translateZ(0.1em);
+  }
+  &:nth-child(5) {
+    transform: translateZ(0.2em);
+  }
+`
+
+function SpinCoin({ src }: { src: string }) {
+  return (
+    <StyledCoin>
+      <StyledCoinFace>
+        <Image src={src} width={60} height={60} />
```

Comment:
no need to optimize the image in this use case. we can use `unoptimized` attribute to disable image optimization.

Reply:
d26a8ae